subschemas/substrate-chain/src/typeDefs/tips.ts

```diff
@@ -9,11 +8,13 @@ export default /* GraphQL */ `
   type Tip {
     "id: Tip Hash"
     id: String!
-    who: AccountInfo!
-    finder: AccountInfo
+    who: Account!
+    finder: Account
     reason: String!
     deposit: String
+    formattedDeposit: String
     closes: String
+    createdAt: String!
     median: String
     formattedMedian: String
```

Comment:
These field names are not really explanatory. 
But I know it doesn't belong to this PR - I was just thinking about naming here 😅  `</rant>`

Reply:
I have changed the field names. 👍